services/ux-backend/main.go

```diff
@@ -87,10 +87,10 @@ func main() {
 		// Set the Deprecation header
 		w.Header().Set("Deprecation", "true") // Standard "Deprecation" header
 		w.Header().Set("Link", "/onboarding/client-tokens; rel=\"alternate\"")
-		clienttokens.HandleMessage(w, r, config.tokenLifetimeInHours, cl, namespace)
+		clienttokens.HandleMessage(r.Context(), w, r, config.tokenLifetimeInHours, cl, namespace)
```

Comment:
You should not use the same context for main and for specific request handling. Each request should initialize its own background context that will be used during the entire request.